minor updates

niyiyu opened the pull request.

niyiyu committed:
Some format editing

Signed-off-by: Yiyu Ni <niyiyu@uw.edu>

niyiyu committed:
made some editing

Signed-off-by: Yiyu Ni <niyiyu@uw.edu>

niyiyu committed:
Update

Signed-off-by: Yiyu Ni <niyiyu@uw.edu>

niyiyu committed:
update

Signed-off-by: Yiyu Ni <niyiyu@uw.edu>

mdenolle commented:
Hi @niyiyu can you review the checks that are still failing? I think it's just the people page to make sure there is a page for each person. rebasing your branch with the head one might fix it.

niyiyu committed:
Merge remote-tracking branch 'origin' into niyiyu/update

niyiyu committed:
Add maleen page

Signed-off-by: Yiyu Ni <niyiyu@uw.edu>

niyiyu commented:
> Hi @niyiyu can you review the checks that are still failing? I think it's just the people page to make sure there is a page for each person. rebasing your branch with the head one might fix it.

Yes I added maleen to the main page without his personal one, and that's the reason the check failed. now added and passed.

niyiyu committed:
ocean actually

Signed-off-by: Yiyu Ni <niyiyu@uw.edu>

niyiyu committed:
Remove duplicates

Signed-off-by: Yiyu Ni <niyiyu@uw.edu>
